enterprise-suite/console-api/prometheus.yml

```diff
@@ -88,6 +88,15 @@ scrape_configs:
         regex: (.*)-[0-9]{13}-driver;
         replacement: ${1}-driver
         target_label: es_workload
+    # flink workload
+      - source_labels: [kubernetes_pod_name, es_workload]
+        regex: (.*)-[a-z,0-9]+-tm;
```

Comment:
It completely depends on how flink labels the pods. Do you have an example of the pod names? It would be good to document it here for future reference.

Reply:
Note this section is for cadvisor metrics only - so CPU/memory usage. Don't we need to add workload extraction in the other sections as well (pod and services)?

Reply:
Yeah I saw that - still worth adding a comment directly in the yml with an example pod name for future reference, I think.

Reply:
Okay, updated.

Let me know if you think the Regex should look for the specific 'hash' length. And can we trust the examples we have to be the way that 'flink labels the pod names' — or is more digging required?

Reply:
I'd suggest taking a look at the flink source code and see how it creates pod names to be certain.